### DISCUSSION TITLE
Sigma rules for syslog/filebeat

### DISCUSSION THREAD

**Discussion by 5MI7th3MI6** [Security-Onion-Solutions/securityonion#9350] [Unsupported Versions] [answered]
Hello, 

Is there a way I can use Playbook to write sigma rules for Syslog? For example, I have a switch that is sending syslogs and I want there to be a way where I can get an alert when specific cmd commands are run. 

![image](https://user-images.githubusercontent.com/68974383/206794458-00b5934a-21c7-4ea3-bd61-2404ef58367f.png)

**Comment by defensivedepth** [comment, marked as answer]
You can do something like this - just make sure to match up the `event.dataset` + `event.module` - Play around with the `convert` functionality until you get the resulting query you need. Also, have you parsed out the `message` field?

![image](https://user-images.githubusercontent.com/954732/207050726-c8e79d6c-0937-4e84-a248-485c76e07b7b.png)

**Reply by 5MI7th3MI6** [reply]
Hi @defensivedepth, 

Thanks for the reply. No, how can I parse the message field?

**Reply by defensivedepth** [reply]
Are you using a Filebeat module to ingest the syslog?

**Reply by 5MI7th3MI6** [reply]
No, these syslogs are coming from a switch so I think it is using port 514 but I am not sure why it says filebeat for metadata.beat? 
![image](https://user-images.githubusercontent.com/68974383/207468983-e6420567-0426-403b-8c63-0fe1d4fabf8f.png)

**Reply by defensivedepth** [reply]
Well, you should be able to still do a wildcard search of the message field

**Reply by 5MI7th3MI6** [reply]
Do you mean on the Dashboard or ssh into the server?

**Reply by defensivedepth** [reply]
In Hunt or Dashboards
